### ISSUE TITLE
Option for default off

### ISSUE THREAD

**Issue by aikruger** [PKM-er/obsidian-floating-toc-plugin#112]
It would be great if there was an ability to set a default of hide of the floating menu in the set up. Great plugin thank you.

**Comment by bootlegcoal** [comment]
Agree! If I understand it right. I would like to hide the menu options on the ToC (pin / search / scroll / copy)
